docs/30-data-valuation.rst

```diff
@@ -23,118 +23,380 @@ interest, but a function of three factors:
 
 pyDVL collects algorithms for the computation of data values in this sense,
 mostly those derived from cooperative game theory. The methods can be found in
-the modules :mod:`~pydvl.shapley` and :mod:`~pydvl.loo`, supported by
+the package :mod:`~pydvl.value`, with support from modules
 :mod:`pydvl.utils.dataset` and :mod:`~pydvl.utils.utility`, as detailed below.
 
+.. warning::
+   Be sure to read the section on
+   :ref:`the difficulties using data values <problems of data values>`.
```

Comment:
Minor nit: I think an `of` is missing between `difficulties` and `using`

Reply:
Why don't you add suggestions? like so:

```suggestion
   :ref:`the difficulties of using data values <problems of data values>`.
```

Reply:
I did not know or think of using it. Thanks

```suggestion
   :ref:`the difficulties using of data values <problems of data values>`.
```